CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
```

Comment:
```suggestion
## 1.1.0
```

Since we're adding new functionality this seems correct to me according to semantic versioning. I'm not really sure if Java projects are typically versioned differently.

Reply:
Agree